solutions/cron/pages/api/cron/[cron].ts

```diff
@@ -0,0 +1,47 @@
+import { NextRequest, NextResponse } from 'next/server'
+
+export const config = {
+  runtime: 'edge',
+}
+
+export default async function handler(req: NextRequest) {
+  const cron = req.nextUrl.searchParams.get('cron')
+  if (!cron) return new Response('No cron provided', { status: 400 })
+  const response = await update(cron)
+  return new NextResponse(JSON.stringify(response), {
+    status: 200,
+  })
+}
+
+async function update(interval: string) {
+  const topstories = await fetch(
+    'https://hacker-news.firebaseio.com/v0/newstories.json?print=pretty'
+  ).then((res) => res.json())
+
+  const response = await fetch(
+    `https://api.vercel.com/v1/edge-config/${process.env.EDGE_CONFIG_ID}/items${
```

Comment:
We could get rid of the EDGE_CONFIG_ID function by using the parseConnectionString(process.env.EDGE_CONFIG) method. That way we only have a single env var that we need to set up. This also then allows us to set up the edge config via the deploy button